dnd-character-generator/src/components/TextBlock.jsx

```diff
@@ -0,0 +1,13 @@
+export default function TextBlock({ label }) {
+    return (
+      <div className="border-2 border-black rounded p-2 bg-gray-50 shadow-inner">
+        <label className="block text-center uppercase text-xs font-semibold mb-2">{label}</label>
+        <textarea
+          rows={4}
+          placeholder="Type here..."
+          className="w-full resize-none p-2 border border-gray-300 rounded focus:outline-none focus:ring-2 focus:ring-yellow-500 text-sm"
+        ></textarea>
+      </div>
+    );
+  }
+  
```

Comment:
Een label heeft een htmlFor attribuut. Als je daar de id van de textarea in plakt dan focused hij de textarea als je op de label klikt.

```suggestion
import { useId } from "react";

export default function TextBlock({ label }) {
  const id = useId();
  return (
    <div className="border-2 border-black rounded p-2 bg-gray-50 shadow-inner">
      <label
        className="block text-center uppercase text-xs font-semibold mb-2"
        htmlFor={id}
      >
        {label}
      </label>
      <textarea
        id={id}
        rows={4}
        placeholder="Type here..."
        className="w-full resize-none p-2 border border-gray-300 rounded focus:outline-none focus:ring-2 focus:ring-yellow-500 text-sm"
      ></textarea>
    </div>
  );
}

```